app/reducers/treatment.js

```diff
@@ -0,0 +1,56 @@
+import axios from 'axios'
+import { browserHistory } from 'react-router'
+import { isBrowser } from '../utils'
+
+const initialState = {
```

Comment:
I think the initial state should just be an empty object. Is there any reason why we would need all these fields initialized?